### DISCUSSION TITLE
Change "Settings" to "Program Settings"

### DISCUSSION THREAD

**Discussion by KodeXall** [keepassxreboot/keepassxc#9300] [Ideas]
(In the wake of this, as it came out to be in the end, totally unnecessary discussion https://github.com/keepassxreboot/keepassxc/discussions/9286)

To avoid stupid mistakes of a user getting hanged up on looking and re-looking for some options in **_Settings_** while totally forgetting about _**Database Settings**_, I hereby propose changing the name of overall options to _**Program Settings**_; or something like _**Settings Of Program**_ or even more descriptive _**Settings of just KeePassXC**_

**Comment by michaelk83** [comment]
It already says "Application Settings" at the top. Maybe just update the title bar to say the same.
Also, I'm not sure which is used more often, but if database settings is more common, then the gear button could default to that when on a database tab. Then application settings could be accessible via a drop-down on that button (plus the main menu).
As it is, the gear button can be confusing for inexperienced users.

**Reply by droidmonkey** [reply]
We aren't going to change anything here. Database Settings is far less common use than application settings.

**Comment by KodeXall** [comment]
How about 2 gear icons sitting next to each other? Each with its own [current] name?

If a user sees two icons, varying even as slightly as being made from the same graphic but one having been rotated by half a tooth, then a user would suspect that it is not a mistake- and that there has to be probably a reason for such duality

Here is a visualization with one of the icons rotated by 90 degrees:

![KeePassXC Settings Icons](https://user-images.githubusercontent.com/107837309/230222011-34f54b65-07b6-44f5-b0d2-69bf183cfa93.jpg)

**Reply by droidmonkey** [reply]
Omg no

**Comment by KodeXall** [comment]
I guess that's a NO no
